packages/expo-modules-autolinking/src/platforms/ios.ts

```diff
@@ -132,3 +135,16 @@ export function formatArrayOfReactDelegateHandler(modules: ModuleDescriptor[]):
   return `[${values.map((value) => `\n${indent.repeat(3)}${value}`).join(',')}
 ${indent.repeat(2)}]`;
 }
+
+async function normalizePodModuleAsync(module: ModuleDescriptor): Promise<string> {
+  let result = module.podName;
+  const podspecFile = path.join(module.podspecDir, `${module.podName}.podspec`);
+  if (await fs.pathExists(podspecFile)) {
+    const { stdout } = await spawnAsync('pod', ['ipc', 'spec', podspecFile]);
```

Comment:
Maybe child_process could do this in a more light-weight way? Doesn't matter much since we'll have it in the CLI anyways

Reply:
i moved spawn-async back to devDependencies and ended up implementing what is basically a lighter-weight version of it in this module's utils [here](https://github.com/expo/expo/pull/16190/commits/cf20021d0843efdd931969edc0151c9ccdb2dc93) (based on child_process, as spawn-async is)